lib/diagnostic.js

```diff
@@ -28,7 +33,13 @@
 // Attributes: name, address, numberOfPets
 
 // Replace null with your answer
-let owner = null;
+let owner = [
+  name: Jacob Hereford
+  age: 30,
+  address: 123 nonya
+  city: Boston
+  numberOfPets: 2
+];
```

Comment:
Your defining this with `[ ]` instead of `{ }` which makes it an array. Also you don't have any comm's seperating each property.